app/controllers/nominations/request_nomination_invite_controller.rb

```diff
@@ -36,7 +36,7 @@ def receive_school
       redirect_to not_eligible_request_nomination_invite_path
     elsif @nomination_request_form.school.registered?
       redirect_to already_nominated_request_nomination_invite_path
-    elsif @nomination_request_form.email_limit_reached?
+    elsif @nomination_request_form.reached_email_limit.present?
```

Comment:
Why do we need to check for `present?`? I would expect the form to give us nice, clean boolean

Reply:
It doesn't - this is why I renamed the method from `email_limit_reached` to `reached_email_limit`. It can be `nil` in most of the cases, but if one of the limits is reached it will return that limit details.